standalone/standalone.py

```diff
@@ -540,6 +541,15 @@ def cli():
     """
 
 
+def validate_pipeline_type(value):
+    if value in ["simple", "full"]:
+        return value
+    elif os.path.exists(value):
```

Comment:
I'm not sure we need this check. How is this handled for other click options? Also, I'm pretty sure `os.path.exists(value):` will fail in all cases since the file you are checking here is what will be called inside of the SDG container not inside the system using the CLI. If this passed it would only be by coincidence.

Reply:
I don't think other click options are doing this sort of a check. But very good point about the location of the callee, I'll remove it.